src/webkit/wkConnection.ts

```diff
@@ -131,8 +131,12 @@ export class WKSession extends platform.EventEmitter {
     const result = new Promise<Protocol.CommandReturnValues[T]>((resolve, reject) => {
       this._callbacks.set(id, {resolve, reject, error: new Error(), method});
     });
-    this._rawSend(messageObj);
-    return result;
+    try {
+      this._rawSend(messageObj);
+      return result;
+    } catch (e) {
+      return Promise.reject(e);
```

Comment:
I'm trying to wrap my head around this. Previously _rawSend was throwing synchronously. Now we reject and return the rejection. How could something not handling the previous rejection start handling the new one?

Aside from my confusion about what this is doing, I think just marking send as async would do the same thing, without needing the try-catch.

Reply:
Done. Changed this to be async.